java/src/main/java/com/logicalclocks/hsfs/engine/Utils.java

```diff
@@ -132,4 +136,35 @@ public String getHiveMetastoreConnector(FeatureGroup featureGroup) throws IOExce
     return connStr + "sslTrustStore=t_certificate;trustStorePassword=" + pw
         + ";sslKeyStore=k_certificate;keyStorePassword=" + pw;
   }
+
+  public Long getTimeStampFromDateString(String inputDate) throws FeatureStoreException, ParseException {
+
+    HashMap<Pattern, String> dateFormatPatterns = new HashMap<Pattern, String>() {{
```

Comment:
I am not completely happy about this, looks hard to maintain. This is something DateTimeFormatter should be able to do out of the box https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html